.vscode/settings.json

```diff
@@ -127,5 +127,7 @@
     "flow.useNPMPackagedFlow": true,
 
     // Clang format styling (duplicated in scripts/CMakeClangUtils.txt)
-    "C_Cpp.clang_format_style": "{BasedOnStyle: Google, BinPackParameters: false, BinPackArguments: false}"
+    "C_Cpp.clang_format_style": "{BasedOnStyle: Google, BinPackParameters: false, BinPackArguments: false}",
+    "prettier.prettierPath": "./GDJS/node_modules/prettier",
```

Comment:
Without that my prettier extension for VS code which I usually use for formatting as I write code instead of running the prettier CLI had a small discrepancy from the CLI, leading to failing format tests. This allows the prettier extension to use the correct version/config thus correctly formatting as expected by the CI.